packages/components/src/CullingInfo/CullingInformation.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { ExclamationCircleIcon, ExclamationTriangleIcon } from '@patternfly/react-icons';
+import { Tooltip } from '@patternfly/react-core';
+import classnames from 'classnames';
+import './CullingInformation.scss';
+import { CullingDate, calculateTooltip, CullingInfo } from './utils';
+export type Render = (config: { msg: string }) => React.ReactElement<any, any> | null;
+export interface CullingInformation {
```

Comment:
As we are passing the `{...props}` to the `Tooltip` component, the interface should extend the `TooltipProps` PF component.

Reply:
Good point!